app/src/main/java/com/android/unio/model/events/EventTypeColor.kt

```diff
@@ -0,0 +1,24 @@
+package com.android.unio.model.events
+
+import androidx.compose.ui.graphics.Color
+
+// class for event types and their corresponding colors
+data class EventTypeColor(val type: String, val color: Color)
```

Comment:
As said in "Event.kt", the "type" attribute should be an Enumeration of type

Reply:
Okay, I will change that ! Thanks